src/classes/CON_Preferred_TEST.cls

```diff
@@ -39,19 +39,6 @@ public with sharing class CON_Preferred_TEST {
 
     @isTest
     public static void preferredEmailPhoneInsert() {
-    
-        Test.testInstall(new STG_InstallScript(), null);
```

Comment:
Running the install script would enable the new functionality, which we actually want to keep disabled. So I just removed this code so that the old code runs.